packages/components/src/Notification/Notification.component.js

```diff
@@ -111,28 +111,67 @@ export function Notification({ notification, leaveFn, ...props }) {
 	);
 }
 
+function Timer(callback, delay) {
```

Comment:
`Timer` and `Registry` are used as classes ; why are you using functions instead of classes ?
I would prefer classes for readability reasons, and I suppose that Babel will do the job anyway.

Reply:
Done